yal/scripts/update_rapidpro_contacts_persona_emoji.py

```diff
@@ -0,0 +1,63 @@
+import argparse
+import json
+
+import emoji
+import requests
+
+from yal.utils import extract_first_emoji
+
+
+def get_arguments() -> argparse.Namespace:
+    parser = argparse.ArgumentParser(
+        description="Pretends to be a webhook from whatsapp for a template message "
+        "button click with payload"
+    )
+    parser.add_argument("url", help="Base URL for the transport")
+    parser.add_argument("api_token", help="Rapidpro API token")
+
+    return parser.parse_args()
+
+
+def update_contacts(arguments: argparse.Namespace):
+    rapidpro_base_api_url = arguments.url
+    api_token = arguments.api_token
+
+    response = requests.get(
+        f"{rapidpro_base_api_url}/api/v2/contacts.json",
+        headers={"Authorization": f"Token {api_token}"},
+    )
+    response = response.json()
+
+    for contact in response["results"]:
```

Comment:
This doesn't seem to handle pagination. I think we'll have to go through all the pages to get all the contacts. The way I like doing pagination is something like:
```python
list_contacts_url = f"{rapidpro_base_api_url}/api/v2/contacts.json"

while list_contacts_url:
   response = requests.get(list_contacts_url, ...).json()
   for contact is response["results"]:
       ...
   list_contacts_url = response["next"]
```